service/bin/geopmd

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+from dasbus.loop import EventLoop
+from dasbus.connection import SessionMessageBus
+import geopmpy.pio
+
+class GEOPMService(object):
+    __dbus_xml__ = """
```

Comment:
Looks like dasbus can also generate the xml for us if we use python3 type annotations and decorate the class with `@dasbus.dbus_interface`. Not sure if there are advantages/disadvantages to that approach though.

Reply:
We will probably need to have access to the XML to write a C++ client, that was my thinking behind creating the interface/io.github.geopm.xml file.

Reply:
TBD in feature branch...This seems like a good feature of dasbus to use.